pkg/kubectl/cmd/cmd_test.go

```diff
@@ -194,6 +194,7 @@ func Example_printMultiContainersReplicationControllerWithWide() {
 		},
 	}
 	cmd.Flags().Set("output", "wide")
+	cmdutil.AddNoHeadersFlags(cmd)
```

Comment:
@soltysh fyi
This test expects human-readable output from the `run` command (output `wide` and `no-headers` flag).
Will update test to expect success message.